<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Song ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Singer </string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>S1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>babythree</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>ngua</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>S2</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>tau hoa nhap ma</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>heo</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>S3</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>khac xuat khac nhap</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>cho</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>S4</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>cay tre tram dot</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>ga</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>S5</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>cong chua ngu trong rung</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>vit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Song ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Song Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Singer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSongID">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSongName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSinger">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
